### ISSUE TITLE
Ternary Statements

### ISSUE THREAD

**Issue by Dealman** [SkaceKamen/vscode-sqflint#71]
SQFLint does not seem to support SQF's equivalent of a ternary statement, example;

```sqf
_angle = if ((_dir1 - _dir2) < 0) then [{(_dir1 - _dir2) * -1}, {_dir1 - _dir2}];
```
Will result in the following error;
`Expected block after then.`

I'm new to VS Code Extensions so if anyone knows where I can modify this behavior myself, that'd be greatly appreciated! 😄

**Comment by SkaceKamen** [comment]
Hey, I think you're not the first to report this. The extension currently only allows this syntax:
```sqf
_angle = if ((_dir1 - _dir2) < 0) then {(_dir1 - _dir2) * -1} else {_dir1 - _dir2};
```

If you want to look into the issue, the extension uses this parser internally:
https://github.com/SkaceKamen/sqflint
which is written in Java

**Comment by Dealman** [comment]
My apologies if it's a duplicate issue, will take a look at that later and see if I can wrap my head around how it works.

Do you think it would be possible to make it so certain errors could be ignored? I think a few other linters out there support something similar.

This is the only false-negative I've run into thus far, so it's working really well otherwise! 👍

**Comment by SkaceKamen** [comment]
Currently, the extension is in maintenance mode, I no longer have time to work on it due to work and personal life.

But I may take a look into this, should be an easy fix

**Comment by veteran29** [comment]
Wasn't @RZSenfo going to take over the maintenance? Just would be nice to know the current state of the extension.

**Comment by RZSenfo** [comment]
Sorry for the late reply, been busy lately.
The underlaying issue is located in the actual sqflint java project.
I will have a look into the project in the next few days and will see what I can do about it

**Comment by SkaceKamen** [comment]
I've already created a fix in the sqflint, I'm just trying to get it properly tested
